app/services/layout.tsx

```
@@ -0,0 +1,70 @@
+                href={`/services/${key}`}
+                className="text-sm font-medium text-muted-foreground transition-colors hover:text-primary"
+              >
+                {category.title}
+              </Link>
+            ))}
+          </div>
+          <button
+            className="inline-flex items-center justify-center rounded-md p-2 text-muted-foreground transition-colors hover:bg-accent hover:text-foreground md:hidden"
+            onClick={() => setIsMenuOpen(!isMenuOpen)}
+            aria-label="Toggle menu"
+          >
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)